### PR TITLE
PE-4310: Error feedback - Pins

### PR DESCRIPTION
**Pull request by matibat**
--- Releases ---
Android release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:android:6cf0cd5ec064fad3ffce07/releases/15u5vb5q305d0

### CODE REVIEW

File: lib/components/pin_file_dialog.dart [hunk 1]
```diff
@@ -152,9 +177,13 @@ class PinFileDialog extends StatelessWidget {
                     child: Align(
                       alignment: Alignment.centerLeft,
                       child: Padding(
-                        padding: const EdgeInsets.only(top: 8.0),
+                        padding: const EdgeInsets.all(8.0),
                         child: Text(
                           customErrorMessage,
+                          style:
+                              Theme.of(context).textTheme.labelLarge!.copyWith(
```
Comment (thiagocarvalhodev) [inline lib/components/pin_file_dialog.dart:184]: You're using the wrong `Theme` class. To use the custom theme from our `ardrive_ui` design system:

```dart
ArDriveTheme.of(context).themeData
```
Maybe the color you want is:

```dart
ArDriveTheme.of(context).themeData.colors.themeErrorDefault
```

The font you might want is:

```dart
ArDriveTypograhpy.body.smallRegular();
```

Comment (matibat) [reply lib/components/pin_file_dialog.dart]: Nice catch! - b56755cce2c40f55fb7c0adb729cf0baadb46911

Comment (matibat) [reply lib/components/pin_file_dialog.dart]: Screenshot 2023-08-14 at 13 49 42

Comment (thiagocarvalhodev) [reply lib/components/pin_file_dialog.dart]: Great!

**Review by thiagocarvalhodev — APPROVED** [review]
LGTM

**Pull request merged**